ocs_ci/ocs/resources/pvc.py

```diff
@@ -70,21 +70,22 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_pvcs(pvc_names):
```

Comment:
I think it would be better if you get here PVC objects instead of names so you won't have to initialize an OCP class inside

Reply:
Done